documentation/API_documentation/Simple_Edge_Discovery_API_Readiness_Checklist.md

```diff
@@ -0,0 +1,13 @@
+# Simple Edge Discovery (EDS) API Readiness minimum criteria checklist
+
+<br>
+
+| No | Deliverables/Criteria            | Mandatory | Status (Contributed/Approved/Pending/Validated/Partly-Validated)|
+|----|----------------------------------|-----------|----------------------------                         
+|  1 |API Spec                          |   Y       | Contributed                |
+|  2 |API Implementation                |   Y       | Contributed                |
```

Comment:
As shown in [Commonalities](https://github.com/camaraproject/Commonalities/blob/main/documentation/API-Readiness-Checklist.md) API implementation is no longer mandatory.